### PR TITLE
fix GPU run for pytorch ignite and lightning example

### PR DESCRIPTION
**Pull request by arisliang**
## Motivation

The pytorch ignite and lightning example failed to run using GPU.

## Description of the changes

This PR fix the run with GPU.

### CODE REVIEW

File: examples/pytorch_lightning_simple.py [hunk 1]
```diff
@@ -149,7 +149,7 @@ def objective(trial):
         val_percent_check=PERCENT_VALID_EXAMPLES,
         checkpoint_callback=checkpoint_callback,
         max_epochs=EPOCHS,
-        gpus=0 if torch.cuda.is_available() else None,
+        gpus=1 if torch.cuda.is_available() else None,
```
[the remark this answers is not in the document]
Comment (arisliang) [reply examples/pytorch_lightning_simple.py:152]: Yes, 0 means no GPU used. So what do you suggest? To change it to `gpus=1 if torch.cuda.is_available() else 0`?

File: examples/pytorch_ignite_simple.py [hunk 2]
```diff
@@ -121,7 +121,7 @@ def log_results(engine):
         "-p",
         action="store_true",
         help="Activate the pruning feature. `MedianPruner` stops unpromising "
-        "trials at the early stages of training.",
+             "trials at the early stages of training.",
```
[the remark this answers is not in the document]
Comment (arisliang) [reply examples/pytorch_ignite_simple.py]: Yes, of course, reverted.

File: examples/pytorch_lightning_simple.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (toshihikoyanase) [reply examples/pytorch_lightning_simple.py:152]: Thank you for your reply. It was just a comment, not a suggestion for change.
`Trainer` accepts `gpus=None` and we can keep your implementation.
https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/trainer/trainer.py#L92

File: examples/pytorch_ignite_simple.py [hunk 3]
```diff
@@ -37,7 +37,6 @@
 
 import optuna
 
-
```
Comment (toshihikoyanase) [inline examples/pytorch_ignite_simple.py:40]: @keisuke-umezawa @hvy @HideakiImamura 
I think we had a coding style convention to put two blank lines after library imports, but I cannot find it in the [current guide](https://github.com/optuna/optuna/wiki/Coding-Style-Conventions). Can we accept this change?

Comment (hvy) [reply examples/pytorch_ignite_simple.py:40]: Oh, I did not know that. We have existing code where there's a single blank line, e.g. similar to this case where variables are declared (e.g. logger objects) or conditions using `try_import`. I thought that the "two blank lines rule" stemmed from PEP-8 https://www.python.org/dev/peps/pep-0008/#blank-lines and that it's okay as long as we follow

> Surround top-level function and class definitions with two blank lines.

Comment (toshihikoyanase) [reply examples/pytorch_ignite_simple.py:40]: @hvy Thank you for your comment.

>  I thought that the "two blank lines rule" stemmed from PEP-8 https://www.python.org/dev/peps/pep-0008/#blank-lines

I understand. I'd like to accept this change according to PEP-8.

**Comment by hvy** [comment]
Let me slightly modify the title to match the release note format https://github.com/optuna/optuna/blob/master/CONTRIBUTING.md#title.

**Pull request merged**